internal/ethereum/deploy_contract_prepare_test.go

```diff
@@ -39,19 +40,91 @@ const samplePrepareDeployTX = `{
 	"gas": 1000000,
 	"nonce": "111",
 	"value": "12345678901234567890123456789",
-	"contract": "0xfeedbeef",
+	"contract": "0xdeadbeef",
 	"definition": [{
 		"inputs": [
 			{
 				"internalType":" uint256",
 				"name": "x",
 				"type": "uint256"
-			}
+			},
+		    {
+			    "internalType":" address",
+			    "name": "y",
+			    "type": "address"
+		    },
+		    {
+			    "internalType":" string",
+			    "name": "z",
+			    "type": "string"
+		    }
```

Comment:
the hole nesting looks weird, probably just GitHub

Reply:
yeah I think that's just github